api/routes/employees.js

```diff
@@ -0,0 +1,55 @@
+const router = require('express').Router()
+const Unit = require('../models/unit')
+
+// This whole thing is ridiculous.
+
+// GET /api/v1/employees?name=[partial-query]
+// GET /api/v1/employees?birthday=[date]
+router.get('/', async (req, res, next) => {
+  const status = 200
+  try {
+    let units = await Unit.find().select('company.employees')
+    let employees = []
+    let employeeMatches = []
+    for (let i = 0; i < units.length; i++) {
+      for (let j = 0; j < units[i].company.employees.length; j++) {
+        employees.push(units[i].company.employees[j])
+      }
+    }
```

Comment:
Another way to do this might be something like:
```js
units.map(unit => unit.company.employees) // Get an array of arrays with employees
     .reduce((prev, curr) => prev.concat(curr)) // Flatten the arrays
```